src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionResolveHandler.cs

```diff
@@ -49,14 +49,16 @@ internal class CodeActionResolveHandler : IRequestHandler<LSP.CodeAction, LSP.Co
         public bool RequiresLSPSolution => true;
 
         public TextDocumentIdentifier? GetTextDocumentIdentifier(LSP.CodeAction request)
-            => ((JToken)request.Data!).ToObject<CodeActionResolveData>().TextDocument;
+            => ((JToken)request.Data!).ToObject<CodeActionResolveData>()?.TextDocument;
```

Comment:
@dibarbet Can't we just put it into our Setup.Dependencies project so that way it's deployed even in main? I would have assumed that'd work fine in this case.